app/views/layouts/application.html.erb

```diff
@@ -12,13 +12,13 @@
   </head>
 
   <body>
-    <%= render partial: 'shared/nav' %>
-    <div class="container">    
-      <% flash.each do |type, message| %>
-      <%= content_tag :div, message, class: type %>
-      <% end %>
-      <%= yield %>
-    </div>
-    <%= render partial: 'shared/footer' %>
+      <%= render partial: 'shared/nav' %>
```

Comment:
This just deletes whitespace. I tried throwing the container in here but the page gets a bit too contained.